pkg/cluster/mpi/mpi.go

```diff
@@ -27,6 +27,15 @@ const (
 	defaultGroupID   = 12574
 	defaultGroupName = "domino"
 
+	// SSH ports used by rsync sidecar
+	rsyncPort     = 2223
+	rsyncPortName = "rsync" // service port name
```

Comment:
Can you change this to `tcp-rsync` for Istio compatibility. 

https://istio.io/latest/docs/ops/configuration/traffic-management/protocol-selection/